acceptance/repo_helper.rb

```diff
@@ -0,0 +1,161 @@
+module PuppetDBExtensions
```

Comment:
Shouldn't this library be a shared one, available via beaker core/some other library?